scripts/update_light_client_checkpoints.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (c) 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+function print_step {
+    echo -e "\e[32m$1\e[0m"
+}
+
+function print_error {
+    echo -e "\e[31m$1\e[0m"
+}
+
+function check_error {
+    if [ $? -ne 0 ]; then
+        print_error "$1"
+        exit 1
+    fi
+}
+
+# Change to the root directory of the repository
+pushd "../"
+
+function cleanup {
+    print_step "Stopping the iota node..."
+    kill -9 $IOTA_PID 2> /dev/null
+
+    if [ $CHANGED_TO_LIGHT_CLIENT_DIR -eq 1 ]; then
+        popd
+    fi
+
+    popd
+}
+
+trap cleanup EXIT
+
+print_step "Build the iota binary in release mode..."
+cargo build --release --bin iota
+check_error "Failed to build the iota binary in release mode"
+
+print_step "Build the iota-light-client 'generate_chk_snapshots' crate in debug mode..."
+cargo build --bin generate_chk_snapshots
+check_error "Failed to build the iota-light-client 'generate_chk_snapshots' crate in debug mode"
+
+print_step "Start the iota node in the background..."
+cargo run --release --bin iota start --force-regenesis --epoch-duration-ms 5000 --with-faucet > /dev/null 2>&1 &
```

Comment:
Does this actually work? I think it needs to look like this:
```suggestion
cargo run --release --bin iota -- start --force-regenesis --epoch-duration-ms 5000 --with-faucet > /dev/null 2>&1 &
```

Reply:
eahm yes it works for me like this on mac 😅 
if I change it to your version, the node doesn't start.

Reply:
That's very strange, because the command those args are being passed to is cargo. For me this does not fail. I think your command is possibly ignoring those arguments?

Reply:
Both work for me on mac 👀

Reply:
tried again, the suggestion doesn't work for me....

Reply:
mhh now it worked... weird. I guess it was an issue with the waiting time? I increased it slightly.